src/components/HeaderWithBackButton.js

```diff
@@ -192,7 +188,7 @@ class HeaderWithBackButton extends Component {
                     )}
                     {this.props.shouldShowAvatarWithDisplay && (
                         <AvatarWithDisplayName
-                            report={this.props.parentReport || this.props.report}
+                            report={this.props.report}
```

Comment:
For sure, i would check everything besides avatar because Georgia is changing it up in a follow up issue. And even if she does not address it i can address it in a follow up so i wouldn't want to block us on the avatar alone.